Languages/es/01_HolaWeb3_es/readme.md

```diff
@@ -0,0 +1,91 @@
+# Tutorial WTF Solidity: 1. HolaWeb3 (Solidity en 3 lineas)
+
+Recientemente, he estado revisando Solidity y escribiendo tutoriales en "WTF Solidity" para principiantes.
+
+Twitter: [@0xAA_Science](https://twitter.com/0xAA_Science) | [@WTFAcademy_](https://twitter.com/WTFAcademy_)
+
+Comunidad: [Discord](https://discord.gg/5akcruXrsk)｜[Wechat](https://docs.google.com/forms/d/e/1FAIpQLSe4KGT8Sh6sJ7hedQRuIYirOoZK_85miz3dw7vA1-YjodgJ-A/viewform?usp=sf_link)｜[Sitio web wtf.academy](https://wtf.academy)
+
+Los códigos y tutoriales están como código abierto en GitHub: [github.com/AmazingAng/WTFSolidity](https://github.com/AmazingAng/WTFSolidity)
+
+-----
+
+## WTF is Solidity?
+
+`Solidity` es un lenguaje de programación utilizado para crear contractos inteligentes en la Ethereum Virtual Machine. Es una habilidad necesaria para trabajar en proyectos  de blockchain. Además, como muchos de ellos son de código abierto, entender el código puede ayudar a evitar proyectos que impliquen pérdida de dinero. 
+
+
+`Solidity` tiene dos características:
+
+1. Orientado a objetos: Después de aprenderlo, puedes usarlo para ganar dinero encontrando los proyectos adecuados. 
+2. Avanzado: Si se escribe un contracto inteligente en Solidity, se es un ciudadano de primer clase de Ethereum.
+
+## Herramienta de desarrollo: Remix
+
+En este tutorial, se usara `Remix` para ejecutar contratos de `solidity`. `Remix` es un IDE (Entorno de desarrollo integrado) de desarrollo de contratos inteligentes recomendado oficialmente por Ethereum. Es adecuado para principiantes, ya que permite la implementación y pruebas rápidas de contratos inteligentes en el navegador, sin necesidad de instalar programas en tu máquina local. 
+
+Sitio Web: [remix.ethereum.org](https://remix.ethereum.org)
+
+Al ingresar a `Remix`, Se puede ver que el menú en el lado izquierdo tiene tres botones, correspondientes a archivo (donde se escribe el texto), compilar (donde se ejecuta el código) y desplegar (donde se despliega en la blockchain). Al hacer clic en el botón "Crear Nuevo Archivo", se puede crear un contrato en blanco de `Solidity`.
+
+
+Dentro de Remix, observamos que hay cuatro botones en el menú vertical a mano izquierda, correspondientes al EXPLORADOR DE ARCHIVOS (donde escribir el código), BUSCAR EN ARCHIVOS (para encontrar y reemplazar archivos), COMPILADOR DE SOLIDITY (para ejecutar código),  DESPLEGAR Y EJECUTAR TRANSACCIONES (para desplegar en la blockchain). Podemos crear un contrato de Solidity en blanco haciendo clic en el botón `Crear nuevo archivo`.
```

Comment:
I think this 2 paragraphs get repeated the information.

Reply:
The English version has the two paragraphs